src/Supports/UtilityHandler.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Bot.Clockify;
+using Bot.States;
+using Luis;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace Bot.Supports
+{
+    public class UtilityHandler : IBotHandler
+    {
+        private readonly LuisRecognizerProxy _luisRecognizer;
+        private readonly DialogSet _dialogSet;
+
+        public UtilityHandler(ConversationState conversationState, LuisRecognizerProxy luisRecognizer)
+        {
+            IStatePropertyAccessor<DialogState> dialogState = conversationState.CreateProperty<DialogState>("UtilityDialogState");
+            _luisRecognizer = luisRecognizer;
+            _dialogSet = new DialogSet(dialogState);
```

Comment:
In this case we have no dialog or am I missing something? How a dialog can be active if this set is empty?

Reply:
it doesn't trigger any multistep dialog, it just sends message activities that are self-conclusive